public/app/features/dimensions/editors/ResourceDimensionEditor.tsx

```diff
@@ -26,7 +26,7 @@ const dummyFieldSettings: StandardEditorsRegistryItem<string, FieldNamePickerCon
 } as any;
 
 export const ResourceDimensionEditor: FC<
```

Comment:
```suggestion
export function ResourceDimensionEditor(props: StandardEditorProps<ResourceDimensionConfig, ResourceDimensionOptions, unknown>) {
```

lets avoid FC when we find them...